packages/main/src/Table.ts

```diff
@@ -160,6 +160,7 @@ type TableRowClickEventDetail = {
 	fastNavigation: true,
 	dependencies: [
 		BusyIndicator,
+		TableHeaderRow,
```

Comment:
@ilhan007 I am still not 100% sure about this change. We are not rendering TableHeaderRow in our hbs file but a table without the header row is not a table since it is required to define the columns of the table. 
Technically it is not a dependency but I am not sure whether we should always consider in our implementation that headerRow[0] might be undefined. In every sample I checked the headerRow[0] was defined even in onEnterDom for the following structure but not in the playground  
```
 <ui5-table>
     <ui5-header-row slot="headerRow">
```

Might this cause any cyclic dependency?

Reply:
Cyclic dependency should not occur here, if we import TableHeaderRow, at least from what I can see.

There is no reference import to Table.ts in TableHeaderRow.ts, TableHeaderCell.ts and TableRowBase.ts.

To be sure, you can also try to build the project locally and use the build in a custom project.

Reply:
@ilhan007 you mentioned that you are discussing with @pskelin. For now I submit the change as having the TableHeaderRow as dependency but if you figure out something please let us know as this dependency IMHO is not a real dependency.